opcua/server/server.py

```diff
@@ -201,54 +207,68 @@ def get_endpoints(self):
 
     def _setup_server_nodes(self):
         # to be called just before starting server since it needs all parameters to be setup
-        self._set_endpoints()
-        self._policies = [ua.SecurityPolicyFactory()]
+        if "None" in self.security_endpoints:
+            self._set_endpoints()
+            self._policies = [ua.SecurityPolicyFactory()]
+
         if self.certificate and self.private_key:
```

Comment:
we should replace that test with a warning. if something else than None is in the list and we are missing private key then we should write a warning to user

Reply:
Something like?
```
if (len(self._security_policy)>1) and self.private_key:
    self.logger.warning("Creating an open endpoint to the server, although encrypted endpoints are enabled.")
```

Reply:
and that one we forgot tp fix

Reply:
weird, on my PC these missing lines (with the additional check) are shown, and yesterday I could see them on GitHub, while now they are missing.